Gemfile.lock

```diff
@@ -418,7 +418,7 @@ GEM
     jquery-rails (3.1.5)
       railties (>= 3.0, < 5.0)
       thor (>= 0.14, < 2.0)
-    jquery-ui-rails (4.0.5)
+    jquery-ui-rails (4.1.2)
```

Comment:
the upgrade is actually to 4.1.2 right?
I guess the gemfile can have 4.1.2 instead of 4.1.1?

Reply:
@luisramos0 Yes. It looks like it is using 4.1.2 even it is told to upgrade to 4.1.1 . I can change this .